.github/workflows/tests.yml

```diff
@@ -0,0 +1,74 @@
+name: "CI"
+
+on:
+  [push, pull_request]
+
+jobs:
+
+  server:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.5.4
+
+      - name: docker-compose
+        run: docker-compose -f .actions-docker-compose.yml up -d
+
+      - name: cache pip
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('dev-requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip
+            ${{ runner.os }}-
+
+      - name: pip install
+        run: |
+            python -m pip install --upgrade pip wheel setuptools
+            pip install -r dev-requirements.txt
+
+      - name: flake8
+        run: flake8
+
+      - name: pytest
+        run: pytest --ignore=tests/aap/ --disable-pytest-warnings --cov=newsroom
+
+      - name: behave
+        run: behave --format progress2 --logging-level=ERROR
+
+  client:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+
+      - name: cache npm
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-npm-${{ hashFiles('package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-npm
+            ${{ runner.os }}-
+
+      - run: npm install --no-audit
```

Comment:
if you have package-lock you can use `npm ci` should be a bit faster

Reply:
ohh right